src/MSALWrapper/MSALWrapper.csproj

```diff
@@ -31,7 +31,7 @@
   <ItemGroup>
     <PackageReference Include="Microsoft.Extensions.Logging.Abstractions" Version="3.1.4" />
     <PackageReference Include="Microsoft.Identity.Client.Extensions.Msal" Version="2.32.0" />
-    <PackageReference Include="Microsoft.IdentityModel.JsonWebTokens" Version="6.16.0" />
+    <PackageReference Include="Microsoft.IdentityModel.JsonWebTokens" Version="7.5.0" />

```

Comment:
Given that this is a major version bump, is there any testing we need to do to ensure this still works as expected? Major version bumps often come with breaking changes.

Reply:
I did our regular testing (mentioned in description). Also had a look at changelog of 7x release, did not find anything that can break anything. Here's the changelog https://github.com/AzureAD/azure-activedirectory-identitymodel-extensions-for-dotnet/wiki/IdentityModel-7x

Reply:
We have an option to do a minor version bump here `6.35.0` which was release in Jan this year https://github.com/AzureAD/azure-activedirectory-identitymodel-extensions-for-dotnet/releases/tag/6.35.0. But 7x has important bug fixes and performance improvement.

Reply:
If we've tested and confirmed it looks good, then I'm more than happy bumping to latest :). Approving!